Gemfile

```diff
@@ -0,0 +1,48 @@
+source 'https://rubygems.org'
+
+ruby '3.3.0'
+
+gem 'bcrypt'
+gem 'bootsnap', require: false
+gem 'cancancan'
+gem 'cocoon'
+gem 'devise'
+gem 'doorkeeper'
+gem 'dotenv-rails'
+gem 'faker'
+gem 'grape'
+gem 'grape-entity'
+gem 'i18n'
+gem 'importmap-rails'
+gem 'jbuilder'
+gem 'jquery-rails'
+gem 'pg'
+gem 'phony_rails'
+gem 'pry'
```

Comment:
I think pry is only used in development environment. Can we move it to development group? Thanks.

Reply:
Yes bhai, we can. Moving it bhai. Thank you.